application/README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://github.com/TheTrain2000/async2rewrite/blob/master/logo.png?raw=true
+    :align: center
+
+.. image:: https://img.shields.io/pypi/v/async2rewrite.svg
+    :target: https://pypi.python.org/pypi/async2rewrite
+.. image:: https://img.shields.io/codecov/c/github/TheTrain2000/async2rewrite.svg
+    :target: https://codecov.io/gh/TheTrain2000/async2rewrite
+
+Disclaimer:
+async2rewrite does not complete 100% of the necessary conversions. It is a tool designed to minimize the amount of tedious work required.
+
+`Migration Documentation`_
+
+.. _Migration Documentation: https://discordpy.readthedocs.io/en/rewrite/migrating.html
+
+About
+------------
+
+.. image:: https://my.mixtape.moe/tphhop.png
```

Comment:
Upload these images inside the directory rather than using some random pomf clone.

I suggest creating a directory such as `images` and placing them all in there.